src/souschef/section.py

```diff
@@ -73,6 +73,5 @@ def value(self, items):
         else:
             self._parent()[self._name] = items
 
-    def update(self, section: Mapping):
-        for k, val in section.items():
-            self[k] = val
+    def update(self, items):
```

Comment:
```suggestion
    def update(self, items: Mapping):
```

Reply:
But when using in grayskull, it is not always a mapping (e.g. `outputs` is a sequence) ...

Reply:
Anyway, I made the change as you suggested.